test/dpul_collections_web/live/home_live_test.exs

```diff
@@ -6,7 +6,7 @@ defmodule DpulCollectionsWeb.HomeLiveTest do
   test "GET /", %{conn: conn} do
     count = DpulCollections.Solr.document_count()
     conn = get(conn, ~p"/")
-    assert html_response(conn, 200) =~ "#{count} Ephemera items"
+    assert html_response(conn, 200) =~ "#{count}\n      Ephemera items"
```

Comment:
I think it would be better if we could not have this newline. Relevant to my earlier comment